package.json

```
@@ -34,6 +35,12 @@
         "title": "Scan for CVEs with Docker Scout",
         "command": "docker.scout.imageScan",
         "enablement": "(view == dockerImages || view == vscode-containers.views.images) && viewItem == image"
+      },
+      {
+        "title": "Build with Debugger",
+        "command": "docker.debug.editorContents",
+        "enablement": "!inDebugMode",
+        "icon": "$(debug-alt)"
```

```suggestion
        "enablement": "!inDebugMode"
```
I was not able to find where this icon is used. If it is not visible anywhere we can drop it for now.